packages/core/app.js

```diff
@@ -1,6 +1,8 @@
 import * as storage from '@octolinker/helper-settings';
 import OctoLinker from './octo-linker.js';
 
+const test2 = 'b';
```

Comment:
<!--
  1 failure:  'test2' is assign...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: packages/core/app.js;
  Line: 4;
-->  
- :no_entry_sign: 'test2' is assigned a value but never used.